modules/mobile/docs/openapi.yaml

```diff
@@ -756,7 +756,136 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Errors"
-
+  /v0/payment_information:
+    get:
+      summary: /v0/payment_information
```

Comment:
Consider naming of this resource. Suffixes like "information" are often just extra. Are these "payments"? "bank_accounts"? "payment_accounts"?

Reply:
I'm with you on the suffixes thing. I did have a short convo with @kreek about an additional object that should be part of this response that makes having the suffix in this case make a bit more sense. Should be included in the yaml now.